library/ssl_tls.c

```diff
@@ -4219,6 +4216,7 @@ int mbedtls_ssl_parse_certificate( mbedtls_ssl_context *ssl )
     size_t i, n;
     const mbedtls_ssl_ciphersuite_t *ciphersuite_info = ssl->transform_negotiate->ciphersuite_info;
     int authmode = ssl->conf->authmode;
+    uint8_t alert;
```

Comment:
Minor issue, I would prefer this to be `unsigned char` as the parameter to `mbedtls_ssl_send_alert_message()`